### PR TITLE
#4487: Fix notification position after sidebar closure

### PR DESCRIPTION
**Pull request by fregante**
## What does this PR do?

- Fixes https://github.com/pixiebrix/pixiebrix-extension/issues/4487

## Discussion

- Requesting review from @BLoe because he recently reviewed similar PRs
- This sets the properties more safely. In the future we might want to rework how the margins are set to avoid overriding any `html` attributes and to allow variable sidebar width

## Demo

gif

## Checklist

- [ ] Add tests
- [ ] Run Storybook and manually confirm that all stories are working
- [x] Designate a primary reviewer: @BLoe  

## Related

- https://github.com/pixiebrix/pixiebrix-extension/pull/4516
- https://github.com/pixiebrix/pixiebrix-extension/issues/4518

### CODE REVIEW

File: src/contentScript/sidebarDomControllerLite.ts [hunk 1]
```diff
@@ -22,15 +22,32 @@
...
+  // Store the original margin so it can be reused in future calculations. It must also persist across sessions
+  html.style.setProperty(
+    ORIGINAL_MARGIN_CSS_PROPERTY,
+    getComputedStyle(html).getPropertyValue("margin-right")
+  );
+
+  // Permanently allow the margin to summed
+  html.style.setProperty(
+    "margin-right",
+    `calc(var(${ORIGINAL_MARGIN_CSS_PROPERTY}) + var(${SIDEBAR_WIDTH_CSS_PROPERTY}))`
```
Comment (fregante) [inline src/contentScript/sidebarDomControllerLite.ts:45]: This splits the two values so they can be updated independently and never have to `parseFloat` the original margin.

Bonus: we can easily update the sidebar width in the future

gif

File: src/contentScript/sidebarDomControllerLite.ts [hunk 2]
```diff
@@ -22,15 +22,32 @@
...
+    return;
+  }
+
+  // Store the original margin so it can be reused in future calculations. It must also persist across sessions
+  html.style.setProperty(
+    ORIGINAL_MARGIN_CSS_PROPERTY,
+    getComputedStyle(html).getPropertyValue("margin-right")
+  );
+
+  // Permanently allow the margin to summed
```
Comment (BLoe) [inline src/contentScript/sidebarDomControllerLite.ts:42]: Sorry, what does this comment mean? Just a little confusing.

**Review by BLoe — APPROVED** [review]
🚀

File: src/contentScript/sidebarDomControllerLite.ts (hunk 2 above)
Comment (fregante) [reply src/contentScript/sidebarDomControllerLite.ts]: _Accidentally a word 😅_

Comments updated

**Pull request merged**